engine/src/test/java/org/camunda/bpm/engine/test/api/cfg/CustomExpressionManagerFunctionsTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.test.api.cfg;
```

Comment:
:wrench: What do you think about moving the test either to `org.camunda.bpm.engine.test.api.cfg` (where `CustomExpressionManagerTest` is) or to it's own package about expression language, e.g. `org.camunda.bpm.engine.test.el` as the test is not related to configuration?

Reply:
I wanted to keep it close to the `CustomExpressionManagerTest`, which is why it's currently in `org.camunda.bpm.engine.test.api.cfg` already (if I am not mistaken?). It also somehow is about configuration (but also EL, yes 🤷) since we want to test that you can configure custom expression manager functions that can then be used (prefixed and not prefixed).

Reply:
My bad on this, I saw an incorrect package at first.